vizro-core/docs/pages/user-guides/card-button.md

```diff
@@ -485,28 +483,48 @@ To create a navigation card, do the following:
           title: Filters and parameters
         ```
     === "Result"
-           [![NavCard]][NavCard]
+           [![NavCardIcon]][NavCardIcon]
 
-       [NavCard]: ../../assets/user_guides/components/nav_card.png
+       [NavCardIcon]: ../../assets/user_guides/components/nav_card_icon.png
 
-If you now click on the card area, you should automatically be redirected to the relevant `href`.
+Note that in the above example the first [`Card`][vizro.models.Card] navigates to an existing [`Page`][vizro.models.Page]
+in the app with `path = filters-and-parameters` and the second one to an external link.
 
-???+ note
+**Make an icon responsive to theme switch**
 
-    When using the [`Card`][vizro.models.Card], keep the following considerations in mind:
+To make an icon responsive to the theme switch, you will need to override the value of the [`filter` CSS property](https://developer.mozilla.org/en-US/docs/Web/CSS/filter).
+The `filter` CSS property lets you add visual effects to elements using different functions. In our example, we're using the `--inverse-color` CSS variable from the Vizro theme.
+It uses the  CSS `invert()` function to flip the color of the icon when you switch themes. Note that this only works if your initial icon has a white fill color. If your icon is not white, you can change its color by adding `fill="white"` to the SVG code.
```

Comment:
> Note that this only works if your initial icon has a white fill color. If your icon is not white, you can change its color by adding `fill="white"` to the SVG code.

Is it worth showing this as an example or linking to somewhere that does?

Reply:
No, I don't think so. This entire section I would actually like to remove from here at some point and add as an example to an "Intro to CSS page". J. asked us to add this section here once, but it's not something that happens frequently.

Reply:
Yeah, sounds good. Let's come back to it when we do the separate CSS page then.

Reply:
Maybe worth making a ticket for a review to tackle Antony's query separately later then?